kube/services/revproxy/gen3.nginx.conf/arborist-service.conf

```diff
@@ -27,6 +27,34 @@ location = /gen3-authz {
     client_max_body_size 0;
 }
 
+#
+# authorization endpoint
+# https://hostname/authz?resource=programs/blah&method=acb&service=xyz
+#
+location ~ /authz/? {
+    error_page 400 =403 @errorworkspace;
```

Comment:
remove these error_page directives,
add in the csrf-check block - ex: https://github.com/uc-cdis/cloud-automation/blob/master/kube/services/revproxy/gen3.nginx.conf/indexd-service.conf#L4